src/appdistribution/client.ts

```diff
@@ -72,17 +72,63 @@ export interface Group {
   inviteLinkCount?: number;
 }
 
+export interface CreateReleaseTestRequest {
+  releaseTest: ReleaseTest;
+}
+
+export enum TestState {
+  IN_PROGRESS = "IN_PROGRESS",
+  PASSED = "PASSED",
+  FAILED = "FAILED",
+  INCONCLUSIVE = "INCONCLUSIVE",
+}
+
+export interface TestDevice {
+  model: string;
+  version: string;
+  locale: string;
+  orientation: string;
+}
+
+export interface DeviceExecution {
+  device: TestDevice;
+  state?: TestState;
+  failedReason?: string;
+  inconclusiveReason?: string;
+}
+
+export interface FieldHints {
+  usernameResourceName?: string;
+  passwordResourceName?: string;
+}
+
+export interface LoginCredential {
+  username?: string;
+  password?: string;
+  fieldHints?: FieldHints;
+}
+
+export interface ReleaseTest {
+  name?: string;
+  deviceExecutions: DeviceExecution[];
+  loginCredential?: LoginCredential;
+}
+
 /**
  * Makes RPCs to the App Distribution server backend.
  */
 export class AppDistributionClient {
-  appDistroV2Client = new Client({
```

Comment:
The `V2` was due to `import { Client } from "../apiv2";` but I think renaming based on the OnePlatform API version probably makes sense